plugin/test/unit/helpers/functions.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { join } from 'path'
+import process from 'process'
+
+import Chance from 'chance'
+import { copy, existsSync } from 'fs-extra'
+import { dir as getTmpDir } from 'tmp-promise'
+
+import { setupImageCdn } from '../../../src/helpers/functions'
+
+const SAMPLE_PROJECT_DIR = `${__dirname}/../../../../demo`
+const TEST_TIMEOUT = 60_000
```

Comment:
Just wondering if there's anyway we can avoid the 60 second timeout?

Reply:
The 60 second timeout was taken from two other tests in this repo (`config.spec.ts` and `utils.spect.ts`). It looks like there must have been an issue with the length of time it takes to copy the demo project in the past, and I'd be hesitant to introduce flakiness.